plugins/modules/vmware_deploy_ovf.py

```diff
@@ -198,6 +198,7 @@
 from ansible.module_utils.urls import generic_urlparse, open_url, urlparse, urlunparse
 from ansible_collections.community.vmware.plugins.module_utils.vmware import (
     find_network_by_name,
+    find_all_networks_by_name,
```

Comment:
```suggestion
from ansible_collections.community.vmware.plugins.module_utils.vmware import (
    find_all_networks_by_name,
```

`find_network_by_name` isn't used anymore, which makes the linter fail:

```
[601] /workspace/src/github.com/ansible-collections/community.vmware$ /workspace/src/github.com/ansible-collections/community.vmware/.tox/linters/bin/flake8 plugins
plugins/modules/vmware_deploy_ovf.py:199:1: F401 'ansible_collections.community.vmware.plugins.module_utils.vmware.find_network_by_name' imported but unused
from ansible_collections.community.vmware.plugins.module_utils.vmware import (
^
ERROR: InvocationError for command /workspace/src/github.com/ansible-collections/community.vmware/.tox/linters/bin/flake8 plugins (exited with code 1)
```